VAMobile/src/App.tsx

```diff
@@ -164,6 +164,9 @@ export const AuthGuard: FC = () => {
   // This is to simulate SafeArea top padding through the header for technically header-less screens (no title, no back buttons)
   const topPaddingAsHeaderStyles = useTopPaddingAsHeaderStyles()
   const [currNewState, setCurrNewState] = useState('active')
+  const screenReaderEnabled = useIsScreenReaderEnabled()
+  const fontScaleFunction = useFontScale()
+  const sendUsesLargeTextScal = fontScaleFunction(30)
```

Comment:
Looks like there might be a typo in the variable name? Not sure what `Scal` stands for at the end.

It'd be ideal if we had a dedicated hook for determining when large text is using, similar to how we have `useIsScreenReaderEnabled`, but that's way beyond the scope of this ticket, and this should work for now